lib/config.ts

```diff
@@ -14,6 +14,10 @@ export class AppConfig {
         enableMultiFactor: false,
         allowedCookieDomains: [],
         skipCookieDomainSecurity: false,
+        enableSingleSignOn: false,
+        singleSignOnUrl: '',
+        singleSignOnEmailAttribute: 'email',
+        singleSignOnScopes: ['openid', 'email'],
```

Comment:
These are the new variables. In most situations, admins should only need to set the first two, but the other two are here to allow for more custom setups.